server/views/shared/actionPlan.njk

```diff
@@ -38,6 +39,10 @@
         {% endif %}
         <h1 class="govuk-heading-l">View action plan</h1>
 
+        {% if presenter.isViewingPreviousVersionOfActionPlan %}
+          {{ govukNotificationBanner(viewingPreviousActionPlanNotificationBannerArgs) }}
```

Comment:
If viewingPreviousActionPlanNotificationBannerArgs was null, would it simply not render like with govukTable?

Reply:
As discussed earlier, we probably don't want to rely on this as it feels like it's relying on nunjucks ignoring a possible error (trying to render null)